src/shell-app-system.c

```diff
@@ -33,6 +33,27 @@
  */
 #define SHELL_APP_IS_OPEN_EVENT "b5e11a3d-13f8-4219-84fd-c9ba0bf3d1f0"
 
+/* Additional key used to map a renamed desktop file to its previous name.
+ * For instance, org.gnome.Totem.desktop contains:
+ *
+ *   X-Endless-Alias=totem
+ *
+ * (without the .desktop suffix).
+ */
+#define X_ENDLESS_ALIAS_KEY     "X-Endless-Alias"
```

Comment:
Iiuc we should drop X-Endless-Alias and only rely on X-Flatpak-RenamedFrom - we already dropped this (X-Endless-Alias) key for most (if not all) our apps that included it.

Reply:
This was on purpose. I didn't want to change any of these patches the day before 3.9.1 freezes.

Reply:
Ok, fair enough, we can drop it later on next rebase if we get to upstream this or some alternative solution.